perf-tests/results/100users-5min-09-03-2022-ec2.md

```diff
@@ -0,0 +1,191 @@
+# Tapir Performance Test
+Number of users: 100
+Time per test: 5min
+Executed on: 09-03-2022
+
+## Setup used
+Two AWS EC2 instances (one running server, one running gatling);
+Intance details: m5.large, 2vCPU, 8GiB RAM, Ubuntu 20.04 LTS;
+Network communication through local IP on a shared EC2 network;
+
+## Summary
+```
+    Name                                     : requests sent
+[1] perfTests.AkkaHttp.TapirMultiServer      :     568554
```

Comment:
for the future, the usual metric that is compared across server impls are requests/second